docs/framework/data/wcf/generating-the-data-service-client-library-wcf-data-services.md

```diff
@@ -8,12 +8,12 @@ helpviewer_keywords:
 ms.assetid: 314077c1-ac10-47e1-bed4-940b5462359d
 ---
 # Generating the Data Service Client Library (WCF Data Services)
-A data service that implements the Open Data Protocol (OData) can return a service metadata document that describes the data model exposed by the OData feed. For more information, see [OData: Service Metadata Document](https://go.microsoft.com/fwlink/?LinkId=186070). You can use the **Add Service Reference** dialog in Visual Studio to add a reference to an OData-based service. When you use this tool to add a reference to the metadata returned by an OData feed in a client project, it performs the following actions:  
+A data service that implements the Open Data Protocol (OData) can return a service metadata document that describes the data model exposed by the OData feed. For more information, see [OData: Service Metadata Document](https://www.odata.org/developers/protocols/overview/#ServiceMetadataDocument). You can use the **Add Service Reference** dialog in Visual Studio to add a reference to an OData-based service. When you use this tool to add a reference to the metadata returned by an OData feed in a client project, it performs the following actions:  
   
 - Requests the service metadata document from the data service and interprets the returned metadata.  
   
     > [!NOTE]
-    > The returned metadata is stored in the client project as an .edmx file. This .edmx file cannot be opened by using the Entity Data Model designer because it does not have the same format an .edmx file used by the Entity Framework. You can view this metadata file by using the XML editor or any text editor. For more information, see the [\[MC-EDMX\]: Entity Data Model for Data Services Packaging Format](https://go.microsoft.com/fwlink/?LinkID=178833) specification  
+    > The returned metadata is stored in the client project as an .edmx file. This .edmx file cannot be opened by using the Entity Data Model designer because it does not have the same format an .edmx file used by the Entity Framework. You can view this metadata file by using the XML editor or any text editor. For more information, see the [\[MC-EDMX\]: Entity Data Model for Data Services Packaging Format](https://www.odata.org/docs/[MC-EDMX].htm) specification  
```

Comment:
```suggestion
    > The returned metadata is stored in the client project as an .edmx file. This .edmx file cannot be opened by using the Entity Data Model designer because it does not have the same format an .edmx file used by the Entity Framework. You can view this metadata file by using the XML editor or any text editor. For more information, see [\[MC-EDMX\]: Entity Data Model for Data Services Packaging Format](https://docs.microsoft.com/openspecs/windows_protocols/mc-edmx/5dff5e25-56a1-408b-9d44-bff6634c7d16).
```